packages/controls/css/labvariables.css

```diff
@@ -30,7 +30,8 @@ all of MD as it is not optimized for dense, information rich UIs.
 /*
  * Optional monospace font for input/output prompt.
  */
-@import url('https://fonts.googleapis.com/css?family=Roboto+Mono');
+ /* Commented out in ipywidgets since we don't need it. */
+/* @import url('https://fonts.googleapis.com/css?family=Roboto+Mono'); */
```

Comment:
The comment seems strange to me ("in ipywidgets"). Also: Why is it commented out and not simply deleted?